fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/CoordinatorServer.java

```diff
@@ -282,6 +285,47 @@ private void registerCoordinatorLeader() throws Exception {
         zkClient.registerCoordinatorLeader(coordinatorAddress);
     }
 
+    private void registerZookeeperClientReconnectedListener() {
+        ZooKeeperUtils.registerZookeeperClientReInitSessionListener(
+                zkClient,
+                () -> {
+                    // we need to retry to register since although
```

Comment:
Also see KAFKA-7165, the ephemeral may still exist. Kafka solve it to drop  existing ephemeral node if the node's session id is less than the current seesion id. 
I think it's kind of complex and hard to understand. So, in here, I just retry when the node exists. Retry 1 minute should work for almost all case.